error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1205,6 +1206,22 @@ Flux<T> after(Flux<? extends Iterable<T>> flux, int prefetch) {
     }
   }
 
+  /**
+   * Don't unnecessarily convert a {@link Collection} to a {@link Stream} before creating a {@link
+   * Flux}.
+   */
```

Comment:
I'll propose a slightly more generic message, that would also fit any additional `@BeforeTemplate` cases.